### DISCUSSION TITLE
Budding Amethyst Silk touch

### DISCUSSION THREAD

**Discussion by Apprisco** [PurpurMC/Purpur#502] [__Completed__] [answered]
Can we get an option to silk-touch budding amethyst blocks?

**Comment by Apprisco** [comment]
Update: nvm, it's already in ToDo for 1.17

**Comment by BillyGalbreath** [comment, marked as answer]
Implemented in build 1281 https://github.com/pl3xgaming/Purpur/commit/d43b5ff644d1ccf981b244d1db36297fe071f2e3
